addons/traefik/metadata.yaml

```diff
@@ -1,5 +1,5 @@
 name: traefik
-version: v2.6.3-1
+version: v2.6.3-2
```

Comment:
Add a dependency addon: gateway-traits

Reply:
The gateway-traits is currently in the expert directory, so I won't add it to it for the time being, or I won't be able to pass the ci.